tests/pfcwd/files/pfcwd_helper.py

```diff
@@ -342,3 +351,16 @@ def start_wd_on_ports(duthost, port, restore_time, detect_time, action="drop"):
     """
     duthost.command("pfcwd start --action {} --restoration-time {} {} {}"
                     .format(action, restore_time, port, detect_time))
+
+
+def fetch_vendor_specific_diagnosis_re(duthost):
+    """
+    Fetch regular expression of vendor specific diagnosis information
+    Args:
+        duthost: The duthost object
+    """
+    unsupported_branches = ['202211']
```

Comment:
I think we are not backporting to 202012 and 202205 as well. Please include those branches as well

Reply:
Hi @neethajohn 
I didn't do that because I see there are dedicated branches for legacy branches. Not it is fixed.